apps/examples/ble_rmc/ble_rmc_main.c

```diff
@@ -70,13 +71,15 @@ static void ble_device_scanned_cb(ble_scanned_device *scanned_device)
 		scanned_device->conn_info.addr.mac[4],
 		scanned_device->conn_info.addr.mac[5]
 	);
+
 	if (g_scan_done == 0) {
-		g_target[0] = scanned_device->conn_info.addr.mac[0];
-		g_target[1] = scanned_device->conn_info.addr.mac[1];
-		g_target[2] = scanned_device->conn_info.addr.mac[2];
-		g_target[3] = scanned_device->conn_info.addr.mac[3];
-		g_target[4] = scanned_device->conn_info.addr.mac[4];
-		g_target[5] = scanned_device->conn_info.addr.mac[5];
+		g_target.mac[0] = scanned_device->conn_info.addr.mac[0];
```

Comment:
How about like below?
`memcpy(g_target.mac, scanned_device->conn_info.addr.mac, sizeof(BLE_BD_ADDR_MAX_LEN)`
We can reduce the code then.

Reply:
Ok, I will use memcpy